.github/workflows/CI.yml

```diff
@@ -68,12 +62,6 @@ jobs:
         password: ${{ secrets.github_token }}
     steps:
     - uses: actions/checkout@v3
-    - name: Set ownership
-      run: |
-        chown -R $(id -u):$(id -g) $PWD 
-    - name: Set ownership
-      run: |
-        chown -R $(id -u):$(id -g) $PWD 
```

Comment:
I don't follow. Why don't we want this for other jobs? If this is in response to https://github.com/chapel-lang/chapel/pull/20965#discussion_r1012945528 my comment was about this call right here that makes 2 chown calls back to back in the same job

Reply:
The error only occurs if we run `git`. The check for tabs script is running `git grep`. Other jobs don't seem to be running `git`.

Reply:
Right, they might not today but they could easily in the future and I don't want developers to have to figure out what this error message means at that time. See also https://github.com/chapel-lang/chapel/pull/20965#discussion_r1012429154

Reply:
IMO the chown is a short-term workaround anyway. This morning @bhavanijayakumaran and I discussed some other approaches. @bhavanijayakumaran could you make that issue so we can share easily what we know?

Reply:
(But it doesn't matter to me if we do or don't do it for the other jobs).

Reply:
Gotcha, yeah I agree with the notion that it's a temporary workaround now, so I'm good with limiting what we use the workaround for.